### PR TITLE
Updated tos, privacy policy, security policy, and contributing files

### PR DESCRIPTION
**Pull request by UnrealApex**
### Description
Made revisions to some files.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,11 +24,11 @@ When contributing to Monkeytype, it's good to know our best practices, tips, and
 
 ### Prerequisites
 
-While most contributions don't require that you install dependencies, there are a few minimum requirements you will need to meet to be able to run the project (this is useful and almost always necessary for tasks like creating features and fixing bugs; running the project is also useful if you are contributing a theme and want to view it on the site before you contribute it). You will need a computer with a stable internet connection, a text editor, Git, Firebase, and NodeJS with a version < 14.
+While most contributions don't require that you install dependencies, there are a few tools you will need to meet to be able to run the project (this is useful and almost always necessary for tasks like creating features and fixing bugs; running the project is also useful if you are contributing a theme and want to view it on the site before you contribute it). You will need a computer with a stable internet connection, a text editor, Git, Firebase, and NodeJS with a version < 14.
```
Comment (Miodec) [inline CONTRIBUTING.md:27]: "few tools you will need to meet" please fix

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,11 +24,11 @@ When contributing to Monkeytype, it's good to know our best practices, tips, and
...
 
 #### Text Editor
 
-If you are not a developer and wish to contribute themes, new languages, or quotes, having a text editor will make contributions _much_ easier. A popular and relatively lightweight editor that we recommend is [Visual Studio Code](https://code.visualstudio.com/) or VS Code. It is free and open-source from Microsoft. Simply run the installer and follow the prompts. Once you have VS Code installed, you are ready to start contributing.
+If you are not a developer and wish to contribute themes, new languages, or quotes, having a text editor will make contributions _much_ easier. A popular choice we recommend is [Visual Studio Code](https://code.visualstudio.com/).
```
Comment (Miodec) [inline CONTRIBUTING.md:31]: Maybe this is a good place to add a section about the github's `.` shortcut to open a web version of vscode?

File: static/privacy-policy.html [hunk 3]
```diff
@@ -83,11 +83,17 @@
       </div>
       <div id="middle">
         <p>
-          This privacy policy explains how Monkeytype uses the data we collect
-          from you when you use our website.
+                  <!-- make sure to update this date every time the policy is changed -->
+          <h3>Effective date: August 27, 2021</h3>
```
Comment (Miodec) [inline static/privacy-policy.html:87]: h3 might not be styled on this page so just use `p` here

File: static/terms-of-service.html [hunk 4]
```diff
@@ -82,13 +82,14 @@
         </div>
       </div>
       <div id="middle">
+        <!-- make sure to update this date every time the tos is changed -->
+        <h3>The last updates made to this terms of service were made on August 27, 2021</h3>
```
Comment (Miodec) [inline static/terms-of-service.html:86]: as mentioned before, use `p`

File: static/terms-of-service.html [hunk 5]
```diff
@@ -131,7 +162,22 @@ <h1>Changes</h1>
...
 
+MONKEYTYPE DOES NOT warrant, endorse, guarantee, or assume responsibility for any user conteNt on the Services or any hyperlinked website or third-party service, and MONKEYTYPE will not be a party to or in any way be responsible for transactions betWeen you and third-parties.
+
+IF APPLICABLE LAW DOES NOT ALLOW THE EXCLUSION OF SOME OR ALL OF THE ABOVE IMPLIED OR STATUTORY WARRANTIES TO APPLY TO YOU, THE ABOVE EXCLUSIONS WILL APPLY TO YOU TO THE FULLEST EXTENT PERMITTED BY APPLICABLE LAW.
+        </p>
```
Comment (Miodec) [inline static/terms-of-service.html:176]: this might need to be divided into separate `p`'s but i don't know

File: static/terms-of-service.html [hunk 6]
```diff
@@ -131,7 +162,22 @@ <h1>Changes</h1>
...
+        <h1>Liability</h1>
+        <p>
+
+        </p>
```
Comment (Miodec) [inline static/terms-of-service.html:180]: empty section

**Review by Miodec — CHANGES_REQUESTED** [review]
Have you tested the new terms and privacy pages locally? Verified they look good?

**Comment by UnrealApex** [comment]
> Have you tested the new terms and privacy pages locally? Verified they look good?

I currently don't have an environment in which I can build and run Monkeytype at the moment, thus most of the PRs I've submitted recently don't require building and running to verify they don't break anything because they're so simple(except for any changes that you need to look at visually).

File: CONTRIBUTING.md (hunk 2 above)
Comment (UnrealApex) [reply CONTRIBUTING.md:31]: Good idea, sure.

**Pull request merged**